hawkbit-core/src/main/java/org/eclipse/hawkbit/api/Base62Util.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2015 Bosch Software Innovations GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.api;
+
+/**
+ * Utility class for Base10 to Base62 conversion and vice versa. Base62 has the
+ * benefit of being shorter in ASCI representation than Base10.
+ *
+ */
+public final class Base62Util {
+    private static final String BASE62_ALPHABET = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+    private static final int BASE62_BASE = BASE62_ALPHABET.length();
+
+    private Base62Util() {
+        // Utility class
+    }
+
+    /**
+     * @param base10
+     *            number
+     * @return converted number into Base62 ASCI string
+     */
+    public static String fromBase10(final Long base10) {
```

Comment:
Is there a need to support `Long`? You'll unbox it already at the next line. BTW even to an integer